mixins/localize-mixin.md

```diff
@@ -121,6 +121,49 @@ If your localized string contains arguments, pass them as a key-value object as
 
 ```javascript
 render() {
-  return html`<p>${this.localize('hello', {firstName: 'Mary'})}</p>`;
+  return html`<p>${this.localize('hello', { firstName: 'Mary' })}</p>`;
 }
 ```
+
+## `localizeHTML()`
+
+You can include some basic markup in your localization resources and safely convert them to HTML with the `localizeHTML()` method
+
+### Simple Markup
+
+#### `[b]Bold[/b]`
+The `[b]` markup converts to the `<strong>` tag
+
+#### `[i]Italic[/i]`
+The `[i]` markup converts to the `<em>` tag
+
+#### `[br]`
+The `[br]` markup converts to the `<br />` tag
+
+
+### Complex markup
+
+Complex markup is more powerful, but requires passing additional information in the second parameter
+
+#### `[a]Linked[/a]`
+The `[a]` markup converts to the `<d2l-link>` tag, and takes an attributes string in the `_links` property
+
+#### `[tt]Tooltipped[/tt]`
+The `[tt]` markup converts to the `<d2l-tooltip-help>` tag, and takes tooltip text in the `_tooltips` property
+
+Complex markup can be used multiple times by including an index and passing an array of its required information
+##### Example
+
+```javascript
+// en.js
+export default {
+  greetingLinks: `Hi {name}, here's [a]a link[/a]. Here's [a 1]another[/a].`
```

Comment:
What's the purpose of the `1` here? Just to identify the link in case they need to be flipped around in the translation?

Maybe we could give them a friendly name identifier similar to how replacements work (e.g. `[a home]` and `[a back]`) and then use those identifiers as keys in an object (instead of an array)?

Reply:
Yep, we can't rely on the English order to fill them in. I like the friendly names/object idea.